services/worker/src/worker/dtos.py

```diff
@@ -218,19 +218,37 @@ class IsValidResponse(TypedDict):
     filter: bool
 
 
-DatasetLoadingTag = Literal["croissant", "webdataset"]
+DatasetTag = Literal["croissant"]
+DatasetLibrary = Literal["mlcroissant", "webdataset", "datasets", "pandas", "dask"]
+ProgrammingLanguage = Literal["python"]
+
+
+class LoadingCode(TypedDict):
+    config_name: str
+    arguments: dict[str, Any]
+    code: str
+
+
+class LoadingMethod(TypedDict):
+    language: ProgrammingLanguage
+    library: DatasetLibrary
+    function: str
+    loading_codes: list[LoadingCode]
 
 
 class DatasetLoadingTagsResponse(TypedDict):
-    tags: list[DatasetLoadingTag]
+    tags: list[DatasetTag]
+    libraries: list[DatasetLibrary]
```

Comment:
isn't libraries redundant with loading_methods, since we get the same info? maybe merge them